.github/workflows/depup.yml

```diff
@@ -32,3 +32,29 @@ jobs:
           branch: depup/reviewdog
           base: master
           labels: "bump:minor"
+
+  staticcheck:
```

Comment:
Since there are too many parameters, it seemed difficult to write them all together using a GitHub Actions matrix 💦 
In this case, It's better to write naively.